docs/parameters.md

```diff
@@ -0,0 +1,53 @@
+
+
+
+The user can provide the parameters below when configuring a storage class to dynamically provision a new FSx for OpenZFS **child volume**.
+
+Aside from ```volumeType```, each of the parameters below map to a corresponding parameter in the FSx API. So, **PLEASE read the documentation for the FSx [create-volume](https://docs.aws.amazon.com/cli/latest/reference/fsx/create-volume.html) API!**
+
+| Parameter                 | Required? | Sample Value                                                   | Notes                                                                                                |
+|---------------------------|-----------|----------------------------------------------------------------|------------------------------------------------------------------------------------------------------|
+| ```volumeType```          | Yes       | "volume"                                                       | Specifies whether the storage class will be used to create an FSx for OpenZFS file system or volume. |
+| ```parentVolumeId```      | Yes       | "fsvol-03062e7ff37662dff"                                      |                                                                                                      |
+| ```copyTagsToSnapshots``` | No        | "false"                                                        |                                                                                                      |
+| ```dataCompressionType``` | No        | "NONE"                                                         |                                                                                                      |
+| ```nfsExports```          | No        | "[{ClientConfigurations=[{Clients=*,Options=[rw,crossmnt]}]}]" |                                                                                                      |
+| ```readOnly```            | No        | "false"                                                        |                                                                                                      |
+| ```recordSizeKiB```       | No        | "128"                                                          |                                                                                                      |
+| ```userAndGroupQuotas```  | No        | "[{Id=1,StorageCapacityQuotaGiB=10,Type=USER}]"                |                                                                                                      |
+| ```tags```                | No        | "Tag1=Value1,Tag2=Value2"                                      |                                                                                                      |
+
+## KEY NOTES:
+* If a parameter is not provided in the storage class, the driver will use the default value specified in the FSx API.
```

Comment:
Please suggest additional notes here. I'm probably missing some key considerations users will need to make when provisioning new PVs